session/pingpong/factory.go

```diff
@@ -38,12 +38,18 @@ import (
 // DefaultAccountantFailureCount defines how many times we're allowed to fail to reach accountant in a row before announcing the failure.
 const DefaultAccountantFailureCount uint64 = 3
 
+// DefaultPaymentInfo represents the default payment info for the alpha release
+var DefaultPaymentInfo = dto.PaymentRate{
+	Price:    money.NewMoney(100000, money.CurrencyMyst),
+	Duration: 1 * time.Minute,
+}
```

Comment:
The same DefaultPaymentInfo is described in services/openvpn/discovery/dto/payment_method_per_time.go
I would expect to have this payment configurations in one package like `payment` etc.